REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.6
+julia 0.7
```

Comment:
this won't work until 0.7.0 final is available

Reply:
The way Pkg3 translates REQUIRE to the new General registry, it actually converts it to the new Pkg3 bounds, so all is well.

Reply:
Hah, that's an interesting accident. The only real concern is that this would not be installable with `OldPkg` on 0.7 but 🤷‍♂️ it's hard to care about that.

Reply:
do new pkg3 bounds handle prerelease julia version bounds at all? or pretend all prereleases are exactly the same as a final release during version resolution?